woocommerce/woocommerce
# Product Collection: On Sale filter allows to "Show only" and "Don't show"

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #49339



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Got a Add New Post
2. Add the Product Collection block
3. Under the Block tab within the Inspector panel locate the Filters section
4. Within the Filters section a new ToogleGroupControl should be shown
<img width="259" alt="image" src="https://github.com/user-attachments/assets/cc32b977-8ccf-4d9b-bf17-52a509b47da5">

5. When `Show only` option is selected the Product Collection block should list on sale products only



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
